stdlib/contrib/anaisdg/anomalydetection/mad.flux

```diff
@@ -65,4 +70,4 @@ mad = (table=<-, threshold=3.0) => {
             )
 
     return output
-}
+}
```

Comment:
Nit: Missing newline

Reply:
Sorry about this. This appears to be applied by the version of the Flux formatter that's installed with the Flux VSCode extension.

Reply:
There seems to have been a discrepancy between the command line formatter and the LSP actually, https://github.com/influxdata/flux/pull/4905 should fix it.